river/base/base.py

```diff
@@ -222,7 +222,7 @@ def find(params):
         return find(self._get_params())
 
     @property
-    def _memory_usage_raw(self) -> int:
+    def _raw_memory_usage(self) -> int:
```

Comment:
Is our only chance relying on `__sizeof__` when using Cython?

Reply:
Yes I think so.